src/ts2lang-app.ts

```diff
@@ -18,4 +18,6 @@ export function run() {
         .parse(process.argv);
 
     main(program["file"]);
-}
\ No newline at end of file
+}
+
+if (!module.parent) { run(); }
```

Comment:
We weren't able to run the project, turns out that it was missing a call to the "run" method. Was it accidentally deleted?